code/controllers/CacheableCacheInspectorController.php

```diff
@@ -124,9 +136,14 @@ private function getTotalCachedObjects($stage, $className = 'CacheableSiteTree')
         if($cache && $siteMap = $cache->get_site_map()) {
             // For reasons unknown, items appear in object-cache with NULL properties
             $cachedSiteTree = ArrayList::create($siteMap)->filterByCallback(function($item) {
-                return !is_null($item->ParentID);
+                if(!is_null($item->ParentID)) {
+                    // push the item ID to the _cachedIDs array for comparison later
+                    $this->_cachedIDs[] = $item->ID;
+                    return true;
+                } else {
+                    return false;
```

Comment:
return false not needed. It's implied by not returning anything. If you still want to keep it, then lose the `else` - it's redundant. Just use `return false` after the first `if` condition.